src/ui/views/Welcome/AccountImport/RecoverPassword.tsx

```diff
@@ -128,9 +128,9 @@ const PasswordIndicator = (props) => {
   );
 };
 
-const SetPassword = ({ handleClick, mnemonic, pk, tempPassword, goEnd, accountKey }) => {
+const SetPassword = ({ handleSwitchTab, mnemonic, pk, tempPassword, goLast, accountKey }) => {
```

Comment:
Component name should match file name. It should be RecoveryPassword not RecoverPassword (or SetPassword)